docs/sql/aggregates.md

```diff
@@ -294,27 +294,6 @@ year | desks | tables | chairs
 (3 rows)
 ```
 
-## Streaming window functions
```

Comment:
Is there a reason why we're removing this section? Probably worth adding a note somewhere about why tumbling windows have to be short intervals.

Reply:
it's deprecated in Calcite, and then we would have a different syntax for HOPPING and TUMBLING windows.